tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -94,12 +167,21 @@ public async Task GetAllAsync_ListOfOrganisationsExist_ReturnsTheOrganisations()
 
             organisationList[0].Name.Should().Be(org1.Name);
```

Comment:
Rather than compare each property, could the code compare object to object? e.g. `organisationList[0].Should().Be(org1)`. This will make the test easier to read by reducing the amount of comparison code.